### PR TITLE
Documentation edi updates

### PR DESCRIPTION
**Pull request by B-yunes**
This branch contains a document where we explain the edi package updates. Since the data format has not yet been standardize, this document only explains general steps for an update, not details about data cleaning.

### CODE REVIEW

File: edi-update-process.Rmd [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    * redd_width_m  
+    * redd_length_m
+
+
+2. Add the Excel file with new data into the `dwr_chinook_redd_survey_data` [folder](https://github.com/FlowWest/feather-redd/tree/main/data-raw/dwr_chinook_redd_survey_data) of your local computer copy, following this naming convention: `yyyy_Chinook_Redd_Survey_Data.xlsx`, replacing "yyyy" with the actual year of the data.
+
+    **Additional Note**: Ensure that the data values are consistent with the expected formats and units. If unsure about the data source or format, consult with the data collection team before proceeding.
+
+## Processing Data
+
```
Comment (alvizek) [inline edi-update-process.Rmd:42]: Is the script called redd_qc.Rmd or redd_qc_2014_2023?

File: edi-update-process.Rmd [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    * percent_small  
+    * percent_med  
+    * percent_large  
+    * percent_boulder  
+    * redd_width_m  
+    * redd_length_m
+
+
+2. Add the Excel file with new data into the `dwr_chinook_redd_survey_data` [folder](https://github.com/FlowWest/feather-redd/tree/main/data-raw/dwr_chinook_redd_survey_data) of your local computer copy, following this naming convention: `yyyy_Chinook_Redd_Survey_Data.xlsx`, replacing "yyyy" with the actual year of the data.
+
```
Comment (alvizek) [inline edi-update-process.Rmd:38]: Clarify this more. If the format is not the same as x file (you could reference last years) then the script wont work

File: edi-update-process.Rmd [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+    clean_names() |>
+    ```
+    ```
+      glimpse()
+    ```
+
+2. After confirming that the data is showing properly, run the `redd_qc.Rmd` script located [here](https://github.com/FlowWest/feather-redd/blob/main/data-raw/redd_qc.Rmd). Note that each year's data was entered differently, so the data cleaning is specific to each year. We will adapt this data cleaning process once the data entry protocol is unified/consistent.
+
+3. The script will generate a CSV called `redd_observations.csv` in the [data folder](https://github.com/FlowWest/feather-redd/tree/main/data).
+
```
Comment (alvizek) [inline edi-update-process.Rmd:61]: change to "update the metadata"

File: edi-update-process.Rmd [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+4. After generating the CSV files, update their metadata (located [here](https://github.com/FlowWest/feather-redd/tree/main/data-raw/metadata)):
+
+    - **Feather Redd** (`feather_redd_metadata.xlsx`): Update the minimum and maximum values for fields such as `date`, `number_redds`, `number_salmon`, etc.
```
Comment (alvizek) [inline edi-update-process.Rmd:64]: If they are adding a new year the time will change. I think remove the "if" statement there and just say "update the..."

File: edi-update-process.Rmd [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+    - If the time range of the data has changed, update the **project metadata** in `feather_redd_project_metadata.xlsx` (under the Coverage tab).
```
Comment (alvizek) [inline edi-update-process.Rmd:65]: Remove "data" before abstract. Can also add methods. So change to "adjust the language in the abstract and methods..."

**Review by alvizek — APPROVED** [review]
This looks good Badhia! I noted a few small items to update. Can you also double check that the numbering makes sense. When I was reviewing in an Rmd it seemed a little confusing so just double check that. After changes are made you can merge into main!

File: edi-update-process.Rmd (hunk 1 above)
Comment (B-yunes) [reply edi-update-process.Rmd:42]: currently markdown is called redd_qc_2014_2023, I think we should modify it to redd_qc, assuming other years will be added. Let me know what you think

**Pull request merged**